meta-mentor-staging/recipes-connectivity/bluez5/bluez5_5.65.bbappend

```diff
@@ -0,0 +1,10 @@
+# Remove patches from meta-imx layer as they fail to apply on bluez 5.65 sources
+SRC_URI_remove = "file://0007-MLK-23858-profiles-audio-increased-the-MTU-size-to-M.patch \
+                  file://0001-bluetooth-Add-bluetooth-support-for-QCA6174-chip.patch \
+"
+
+# Apply devtool adjusted patches from meta-imx layer 
+# prepend here so that other patches from meta-imx layer can be applied cleanly
+SRC_URI_prepend = "file://0001-imx_layer_adjusted_PATCH-MLK-23858-profiles-audio-increased-the-MTU-siz.patch \
+                   file://0001-imx_layer_adjusted_PATCH-bluetooth-Add-bluetooth-support-for-QCA6174-chip.patch \
+"
```

Comment:
I won't agree to this approach.... we probably need to provide these patches directly from meta-mentor-cve (imperfect) using a layer specific dir or through the imx bsp common (ideally). The point is, this is imx specific and not applicable to every bsp.

Reply:
agreed. created https://github.com/MentorEmbedded/meta-mentor-bsp/pull/1244